### ISSUE TITLE
Question: you need 2 dollars and you have a source code here??!!

### ISSUE THREAD

**Issue by binary-person** [kzahel/jstorrent-legacy-app#144]
hi, its me again, just wondering that you put 2.99 dollars on chrome store?

**Comment by kzahel** [comment]
Yes source code is provided for "educational" reasons. It's pretty common to have open source software and still monetize through various means. Look at gitlab for example, an open source version of github.

**Comment by kzahel** [comment]
The new JSTorrent is completely free — no purchases needed. Get it at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues, please leave feedback!
